common/common.go

```diff
@@ -271,6 +276,31 @@ func getBlockFromRPC(height int) (*walletrpc.CompactBlock, error) {
 		return nil, errors.New("received unexpected height block")
 	}
 
+	// Get the txids; it would be more efficient to implement a
+	// getblock verbose level that returns both raw hex and the txid list
+	// so that this second rpc wouldn't be needed.
```

Comment:
This comment doesn't explain why this is necessary, which is that `block.ParseFromSlice` is broken for blocks containing v5 transactions.
```suggestion
	// `block.ParseFromSlice` correctly parses blocks containing v5 transactions, but
	// incorrectly computes the IDs of the v5 transactions. We temporarily paper over this
	// bug by fetching the correct txids via a second getblock RPC call.
	// https://github.com/zcash/lightwalletd/issues/392
```

Reply:
This comment still needs rewriting, at a minimum to reference the open issue, but ideally saying that this second fetch is necessary because `block.ParseFromSlice` doesn't yet implement txid derivation.